_layouts/activity-default.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html lang="{{ page.lang }}">
+
+  {% include head.html %}
+
+  <body>
+
+    {% include header.html %}
+
+    <div class="page-content">
+      <div class="wrapper activity-page">
```

Comment:
from what I remember without checking, the activity-page class is the only thing that changes from the default layout. Could you check if there is any other way how we could inject that so we don't need to do a separate file?